azext_iot/_params.py

```diff
@@ -382,7 +382,7 @@ def load_arguments(self, _):
             help="Blob Shared Access Signature URI with write, read, and delete access to "
             "a blob container. This is used to output the status of the "
             "job and the results. Note: when using Identity-based authentication, you must supply an "
```

Comment:
I think this makes the last part of the text more clear (same application for all the descriptions) 

"Note: when using Identity-based authentication an https:// URI is still required. Input for this argument can be inline or from a file path."